tools/mo/requirements_onnx.txt

```diff
@@ -1,8 +1,8 @@
-onnx>=1.8.1,<=1.12
-networkx~=2.5; python_version <= "3.6"
-networkx<=2.8.8; python_version > "3.6"
+-c ../constraints.txt
 numpy>=1.16.6,<1.25.0
-defusedxml>=0.7.1
-requests>=2.25.1
-fastjsonschema>=2.15.1,<2.17
-protobuf>=3.18.1,<4.0.0
+onnx
+networkx
+defusedxml
+requests
+fastjsonschema
+protobuf
```

Comment:
Some tests directly install these files like `pip install -r requirements_onnx.txt` that means they will ignore versions?
Please check this one, for example: https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/linux.yml#L165

Reply:
Pip understands `-c` flag, so it will use versions from `openvino/tools/constraints.txt`. It can be seen in latest `openvino-lin` job at https://dev.azure.com/openvinoci/dldt/_build/results?buildId=481344&view=logs&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=83ea7ca3-25f9-5f6d-cb63-d032d85612bf, in the "Install dependencies" step that the versions installed are correct.

Reply:
can we mark this conversation as solved and move forward? :)

Reply:
there are a lot of failires in CI for this PR. Propose to anticipate fixing it and we will resolve it:)